### DISCUSSION TITLE
MemoryError everytime

### DISCUSSION THREAD

**Discussion by zhouyi0812** [flatironinstitute/CaImAn#1280] [Q&A]
Hello!
Recently each time I run cnmfe, there is an memory error issue. The same file, the same parameter which was successfully run before doesn't work now. 
I follow the latest caiman demo and adjust according to my data. I don't understand that why it requires so much memory. 
And it doesn't work with restart the kernel and computer. And usually it broken during the merging step.

Thank you very much!

![image](https://github.com/flatironinstitute/CaImAn/assets/48576330/c708fa8f-d3c1-4781-8305-6a3ae7190e45)
![image](https://github.com/flatironinstitute/CaImAn/assets/48576330/d39f8f09-dbc1-4ba9-917c-797b5fd579ca)

**Comment by EricThomson** [comment]
If you run the standard cnmfe demo notebook, with the demo data, are you getting this error? If so, at which step exactly does the problem appear?

Note there will be a new release this week I would install that when it comes out (1.10.3) and see if that helps (it probably won't but it might).

**Reply by zhouyi0812** [reply]
Thank you for your reply!!
I upgrade the version, but it is still not working, I wonder if there is a way that I can debug if it is a problem from my computer? Since it works well before.
Thank you!
